app/assets/stylesheets/views/_covid.scss

```diff
@@ -141,6 +141,10 @@
     @include govuk-media-query($from: tablet) {
       margin-bottom: govuk-spacing(3);
     }
+
+    .covid__list-item-description--featured {
```

Comment:
Do we need to nest this class like this? We generally avoid nesting with BEM if we can.